### PR TITLE
Copied Brought Forward Losses page for shares

### PR DESCRIPTION
**Pull request by morgan-fernandes**

### CODE REVIEW

**Comment by DavidWoodward37** [comment]
Hi morgan. Sorry these are probably phantom errors but could you merge master again?

File: app/controllers/resident/shares/DeductionsController.scala [hunk 1]
```diff
@@ -35,7 +35,6 @@ import play.api.data.Form
 import views.html.calculation.{resident => commonViews}
 import views.html.calculation.resident.properties.{deductions => views}
```
Comment (jameshforster) [inline app/controllers/resident/shares/DeductionsController.scala]: needs to be changed to shares

File: app/views/calculation/resident/lossesBroughtForward.scala.html [hunk 2]
```diff
@@ -16,7 +17,7 @@
 
     <h1 class="heading-large">@Messages("calc.resident.lossesBroughtForward.question", taxYear.taxYearSupplied)</h1>
 
-    @form(action = controllers.resident.properties.routes.DeductionsController.submitLossesBroughtForward) {
+    @form(action = controllers.resident.shares.routes.DeductionsController.submitLossesBroughtForward) {
```
Comment (jameshforster) [inline app/views/calculation/resident/lossesBroughtForward.scala.html]: Needs to have a passed in value to distinguish between properties and shares.

File: test/controllers/resident/shares/DeductionsControllerSpec/LossesBroughtForwardActionSpec.scala [hunk 3]
```diff
@@ -0,0 +1,320 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.resident.shares.DeductionsControllerSpec
+/*
```
Comment (jameshforster) [inline test/controllers/resident/shares/DeductionsControllerSpec/LossesBroughtForwardActionSpec.scala]: Double header

File: test/routes/shares/RoutesSpec.scala [hunk 4]
```diff
@@ -71,6 +71,14 @@ class RoutesSpec extends UnitSpec with WithFakeApplication with Matchers {
...
   }
 
+
+  "The URL for the lossesBroughtForward action" should {
+    "be equal to /calculate-your-capital-gains/resident/shares/losses-brought-forward" in {
+      val path = controllers.resident.shares.routes.DeductionsController.lossesBroughtForward.toString()
+      path shouldEqual "/calculate-your-capital-gains/resident/shares/losses-brought-forward"
+    }
+  }
+
```
Comment (jameshforster) [inline test/routes/shares/RoutesSpec.scala]: Needs test for post route.

**Pull request merged**